### PR TITLE
Who we are

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Presents the organization

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
-# who-we-are
-This repository presents this organization
+# Friends of PrestaShop
+
+This repository presents Friends of PrestaShop community, its missions, intents and publics.
+
+## Publics
+
+We are PrestaShop integrators and developers, working together on modules themes and additional resources.
```
Comment (sarjon) [inline README.md]:
```suggestion
We are PrestaShop integrators and developers, working together on modules, themes and additional resources.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
+
+> Is this organization and its differents channels open for merchants and complete beginners?
+
+No, please refers to the official forum of PrestaShop if you're a merchant or to the official User or Developers documentations if you needs to learn PrestaShop basics.
```
Comment (sarjon) [inline README.md]:
```suggestion
No, please refers to the official forum of PrestaShop if you're a merchant or to the official User or Developers documentations if you need to learn PrestaShop basics.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,25 @@
-# who-we-are
-This repository presents this organization
+# Friends of PrestaShop
+
+This repository presents Friends of PrestaShop community, its missions, intents and publics.
+
+## Publics
```
Comment (sarjon) [inline README.md:5]: I don't get the meaning of this. :thinking:

File: README.md [hunk 4]
```diff
@@ -1,2 +1,25 @@
...
+
+We are PrestaShop integrators and developers, working together on modules themes and additional resources.
+
+> Is this organization and its differents channels open for merchants and complete beginners?
+
+No, please refers to the official forum of PrestaShop if you're a merchant or to the official User or Developers documentations if you needs to learn PrestaShop basics.
+
+## Intents
```
Comment (sarjon) [inline README.md:13]: Maybe `Focus`? Not sure though.

**Review by sarjon — COMMENTED** [review]
:+1:

**Review by jf-viguier — COMMENTED** [review]
We should use "Friends of Presta" instead of "Friends of Prestashop" ?

File: README.md (hunk 3 above)
Comment (mickaelandrieu) [reply README.md:5]: hmm, it's more like "people we accept in the community"

=> Concerned public instead?

File: README.md (hunk 4 above)
Comment (mickaelandrieu) [reply README.md:13]: maybe something easier for everyone: "What we do"

wdyt ?

Comment (sarjon) [reply README.md:13]: :ok_hand:

File: README.md (hunk 3 above)
Comment (sarjon) [reply README.md:5]: "People behind the project" or simply "People"?

File: README.md (hunk 4 above)
Comment (jf-viguier) [reply README.md:13]: ok

File: README.md (hunk 3 above)
Comment (jf-viguier) [reply README.md:5]: Go for People

**Comment by kpodemski** [comment]
great 👍

**Pull request merged**